phpBB/includes/acp/acp_profile.php

```diff
@@ -672,7 +672,12 @@ function main($id, $mode)
 				$s_one_need_edit = true;
 			}
 
+			if (!isset($this->type_collection[$row['field_type']]))
+			{
+				continue;
+			}
```

Comment:
This change has already been merged by https://github.com/phpbb/phpbb/pull/4555, so remove to avoid merge conflicts and duplicate code.